dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -16,11 +16,30 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+# tflint-ignore: terraform_naming_convention
+resource "kafka_topic" "RatingEvents_v3" {
+  name = "cbc.RatingEvents_v3"
+
+  replication_factor = 3
+  partitions         = 15
```

Comment:
Is this a migrated topic? If not, can it manage with less partitions?

Reply:
it is a migrated topic yeah, we'll be migrating a bunch of them in the next few days